app/src/main/java/sdp/moneyrun/MenuActivity.java

```diff
@@ -222,6 +222,14 @@ public void onButtonSwitchToUserProfileActivity(View view) {
         setPutExtraArguments(playerProfileIntent);
         startActivity(playerProfileIntent);
     }
+
+
```

Comment:
Maybe remove comments? Or will you use this later?

Reply:
I didnt write that code, it must be the result of a bad merge. I removed it